.github/workflows/regression_test.yml

```diff
@@ -22,6 +22,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cpu
```

Comment:
cc @cpuhrsch @msaroufim

Reply:
There's another job right below (`test-nightly`) that does this

Reply:
I guess I could make this work by checking pytorch versions, do you think that's what I should do?

Reply:
this is done, please take a look again